tests/sentry/integrations/discord/test_client.py

```diff
@@ -0,0 +1,130 @@
+import responses
+from django.test import override_settings
+
+from sentry import options
+from sentry.integrations.discord.client import DiscordClient
+from sentry.silo.base import SiloMode
+from sentry.silo.util import PROXY_BASE_PATH, PROXY_OI_HEADER, PROXY_SIGNATURE_HEADER
+from sentry.testutils.cases import TestCase
+
+
+class DiscordClientTest(TestCase):
+    guild_name: DiscordClient
```

Comment:
Should this be `discord_client`?

Reply:
@spalmurray  I don't see `self.guild_name` anywhere here.

Reply:
Good catch 😆